pkgs/by-name/ya/yafc-ce/package.nix

```
@@ -12,13 +12,13 @@ let
 in
 buildDotnetModule (finalAttrs: {
   pname = "yafc-ce";
-  version = "2.14.0";
+  version = "2.16.0";
 
   src = fetchFromGitHub {
     owner = "shpaass";
     repo = "yafc-ce";
-    rev = finalAttrs.version;
-    hash = "sha256-EbVPPSci4AziF+n/8pIa9M44WEUj+vvEOQqu3HJ6aNQ=";
+    rev = "v${finalAttrs.version}";
```

Since you explicitly mentioned they are using tags:


```suggestion
    tag = "v${finalAttrs.version}";
```

This prevents accidentally checking out a branch with the same name.